compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -571,12 +571,15 @@ extern llvm::cl::opt<std::string> clGPUCodegenTransformDialectFileName;
 extern llvm::cl::opt<std::string> clGPUCodegenTransformDialectDebugPayloadTag;
 extern llvm::cl::opt<std::string> clGPUCodegenTransformDialectDebugTransformTag;
 
-void addGPUTransformDialectPasses(OpPassManager &passManager) {
+void addGPUTransformDialectPasses(OpPassManager &passManager,
+                                  StringAttr codegenSpecFileName) {
   // Give control to the transform dialect.
+  StringRef fileName = codegenSpecFileName == StringAttr()
+                           ? clGPUCodegenTransformDialectFileName
+                           : codegenSpecFileName.getValue();
   passManager.addPass(
       mlir::iree_compiler::createTransformDialectInterpreterPass(
-          clGPUCodegenTransformDialectFileName,
-          clGPUCodegenTransformDialectDebugPayloadTag,
+          fileName, clGPUCodegenTransformDialectDebugPayloadTag,
```

Comment:
If we match the order of the cpp file we'll need something like:
```suggestion
          fileName, /*transformLibraryFileName=*/StringRef(),
          clCPUCodegenTransformDialectDebugPayloadTag,
```